app/views/layouts/_header.html.erb

```diff
@@ -1,18 +1,25 @@
 <header>
-  <%= image_tag("Logo Blue.jpg", class: "logo") %>
-  <h1> Any Rat Rescue </h1>
-  <nav class="menu">
-    <ul>
+  <div class="banner-container">
+    <div class="banner">
+      <%= image_tag("Logo Blue.jpg", class: "logo") %>
```

Comment:
I didn't notice this before but in general it's good practice to keep filename lowercase and without spaces. I'd rename this file in `app/assets/images` to `logo_blue.jpg` and then call it here. You're not really doing anything wrong here, BUT filesystems treat uppercase letters differently (some ignore them and some don't) so it's generally best practice to stick with lower case for these types of things.